ui/js/controllers/perf/compare.js

```diff
@@ -307,76 +389,78 @@ perf.controller('CompareResultsCtrl', [
                 });
         }
 
+        function updateURL() {
+            $state.transitionTo('compare', {
+                framework: $scope.filterOptions.framework.id,
+                filter: $scope.filterOptions.filter,
+                showOnlyImportant: $scope.filterOptions.showOnlyImportant ? undefined : 0,
+                showOnlyConfident: $scope.filterOptions.showOnlyConfident ? 1 : undefined
+            }, {
+                location: true,
+                inherit: true,
+                relative: $state.$current,
+                notify: false
+            });
+        }
+
         $scope.dataLoading = true;
 
         var loadRepositories = ThRepositoryModel.load();
         var loadFrameworks = PhFramework.getFrameworkList().then(
             function(frameworks) {
                 $scope.frameworks = frameworks;
             });
+
         $q.all([loadRepositories, loadFrameworks]).then(function() {
-            $scope.errors = PhCompare.validateInput($stateParams.originalProject,
-                                                    $stateParams.newProject,
-                                                    $stateParams.originalRevision,
-                                                    $stateParams.originalProject);
+            $scope.errors = [];
+            /*$scope.errors = PhCompare.validateInput($stateParams.originalProject,
+                                            $stateParams.newProject,
+                                            $stateParams.originalRevision,
+                                            $stateParams.newRevision);
 
```

Comment:
PhCompare.validateInput will not work for base line comparison since we our nor passing originalRevision. This is what is causing a merge conflict. Should I define a new function similar to validate input here:

https://github.com/SJasoria/treeherder/blob/Bug_1273513/ui/js/services/perf/compare.js#L191